### DISCUSSION TITLE
New Release

### DISCUSSION THREAD

**Discussion by vladmandic** [vladmandic/sdnext#2448] [Announcements]
Changelog as usual: https://github.com/vladmandic/automatic/discussions/99#discussioncomment-7492577

**Comment by a2ZxgdqHeSs8F51cIyuK** [comment]
Hello, not sure if this is a bug or just me missing something, so I figured I'd ask here first (I can't open Discord on my PC, takes 15+ minutes to open, then 5+ minutes to do anything. No idea why).
So, big update. Doesn't work.
So I moved away the extensions (both built in and user). Now it works.
Added my user extensions back in, found one causing issues. Wasn't even one I cared about so all good, with the recreated builtin extensions and the trimmed user extensions it starts.
However now LoRas and Lycos don't work anymore. They don't show up in the Networks menu either. 
Did something change? How do I get LoRas and Lycos on the new version? There used to be a built in "extra networks" extension but it seems to be gone now...
My auto complete can still see/suggest my LoRas but they don't actually do anything.
Should I open an issue for this?
EDIT: The new "built-in styles" don't seem to do anything either.
EDIT 2: Under "Settings->Extra networks->Extra networks UI->Extra networks" I have a few options but no LoRas and no Lycos.

**Reply by vladmandic** [reply]
Lyco handler doesn't exist anymore, it's a. Under unified lora handler. And if you have lyco extension, remove it.

And they all show under extra networks - lora.

**Reply by a2ZxgdqHeSs8F51cIyuK** [reply]
I don't have any extensions specifically for Lyco or LoRa.
I don't have either of them under Extra Networks, unless I'm looking in the wrong place? 
Here's what I see: https://imgur.com/a/TN7j2w5

**Reply by vladmandic** [reply]
Your install doesn't seem healthy. I don't know how you did it, but I'd suggest a clean install as I can't go over every possibility of corrupt install.

**Reply by a2ZxgdqHeSs8F51cIyuK** [reply]
That is terrible.
Going to have to spend hours tomorrow reinstalling it, moving things over from the backup one by one, checking every time, then probably go mad every 10 minutes when I find something that isn't how I customised it...
I tried safe mode and it has the same behaviour: no LoRas. No errors either.

**Reply by vladmandic** [reply]
Move what? Models? You can point to old folders in settings. Don't need to move anything.

**Reply by a2ZxgdqHeSs8F51cIyuK** [reply]
Configurations and customisations mostly. I'm probably exaggerating it, I'm just a bit tired of everything at the moment.
Models will be pretty easy to move, just need to remember everything. Extensions, LoRas, Lycos, upscalers, etc.
EDIT: So I got a basic install going. Moved over just the models folder for now. LoRas show up now. Lycos don't show up as a category, but the models do appear mixed in with the LoRas.
EDIT 2: I think I got most of it set up how I want it. How do I disable live previews in this new version? I can't find the setting for turning them off. Back on the old versions I ran some tests and gens were very significantly slower with live previews active.

Another question: Is there a way to configure, on the TXT2IMG and IMG2IMG "tabs", which menus are open or closed by default?
Right now on both pages "sampler" is open by default. I'd like to have nothing open on TXT2IMG and Resize and Denoise open for IMG2IMG. Or if they can't be different, just have them all start closed by default.
I checked in the settings but didn't see it, might have missed it.

**Reply by anothertal3** [reply]
Setting `Live preview display period` to -1 or 0 should disable live previews, afaik.

**Reply by a2ZxgdqHeSs8F51cIyuK** [reply]
> Setting `Live preview display period` to -1 or 0 should disable live previews, afaik.

Ah you're right. I did change that, but hadn't done a full shutdown and restart of the WebUI.
EDIT: Another issue, I'm on the default theme for this version.
In inpainting the slider for choosing the size of the "brush" is unusably tiny: https://imgur.com/OGWFHvZ
That's all the way to the left, the tiny grey sliver to the right is the whole range of movement for the selector.

**Reply by vladmandic** [reply]
> Another question: Is there a way to configure, on the TXT2IMG and IMG2IMG "tabs", which menus are open or closed by default?

go to ui settings and save your defaults.

> In inpainting the slider for choosing the size of the "brush" is unusably tiny:

haven't noticed, i'll check.

**Reply by a2ZxgdqHeSs8F51cIyuK** [reply]
> go to ui settings and save your defaults.

I use that function extensively (very good addition, it's from the previous big update I think? That's when I noticed it anyway) but it doesn't seem to have memory for which menus are open or collapsed. Doesn't show up in the list of things it would save.

**Reply by vladmandic** [reply]
there is literally a button "set menu states"
![image](https://github.com/vladmandic/automatic/assets/57876960/c5a0dd53-6183-441a-98d2-e6c240f5fdfb)

**Reply by a2ZxgdqHeSs8F51cIyuK** [reply]
Another issue I'm really noticing now that I'm trying to use it.
Something is very wrong with IMG2IMG.
TXT2IMG seems fine (it's hard to tell since my initial outputs are fairly ugly to begin with but that's normal with my procedure). I usually take my outputs to IMG2IMG where I refine them, upscale them, make edits, etc.
Outputs from IMG2IMG look absolutely awful.

Here's some super simple proof, with a simple prompt.
https://imgur.com/a/xUTX2nK
In order:
TXT2IMG output. Looks fine for a first output. Ignore mistakes like the hands or trees, this is just a quick output to illustrate the issue. Focus on the face.
I take the image to IMG2IMG. Same prompt. Same everything. Same size, denoise 0.4. Notice the significant loss of detail, it's the opposite of what should be happening.
I do it again. More detail loss.
Again. It looks awful.

Edit: Oh, I must have missed that about menu states. Nifty.
I've just been spending the whole day on a bunch of different stuff, including trying to understand why all my images look like absolute garbage now, I'm a bit out of sorts.

**Reply by vladmandic** [reply]
that "simple proof" is not actionable. create issue, upload actual txt2img and img2img. and settings for both.

**Reply by a2ZxgdqHeSs8F51cIyuK** [reply]
Didn't realize Imgur strips the metadata, garbage site. I'll open an issue and use catbox instead.

**Reply by vladmandic** [reply]
just upload images to github issue directly.

**Reply by Tillerz** [reply]
Hm, where is that "extra networks -> reference", I cannot find that. That should be in System / Extra Networks, right?

**Reply by vladmandic** [reply]
extra networks -> models -> reference, but you must be in diffusers mode for it to show up.

**Reply by Tillerz** [reply]
Ok, that was the issue then. :)
